code-security/admin_guide/get-started/setup-administrator-access.adoc

```diff
@@ -7,7 +7,9 @@ To know more see https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cl
 
 Administrators can create a custom permission group for Code Security on the Prisma Cloud console. Using the parameters for permissions, you can limit or enhance the responsibilities of the users.
 
-* *Suppression Management*: You can enable user permissions to view, create, update and delete resources on *Repositories* (Settings > Repositories). These parameters enable suppression management for vulnerabilities found in resources. In addition, configuring view permission for all Code Security functions allows you to see the resource vulnerabilities to make informed decisions.
+* *Repositories*: You can enable user permissions to view, create, update and delete resources on *Repositories* (Settings > Repositories). These parameters enable suppression management for vulnerabilities found in resources. In addition, configuring view permission for all Code Security functions allows you to see the resource vulnerabilities to make informed decisions.
```

Comment:
remove this line please - it's irrelevant

https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin/manage-prisma-cloud-administrators/prisma-cloud-admin-permissions

Reply:
Tracking the change in https://redlock.atlassian.net/browse/RLP-99394[RLP-99394] @GiladMark